test/raft.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Raft } from '../typechain-types'
+import { Event, Wallet } from 'ethers'
+import { waffle } from 'hardhat'
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers'
+
+describe('Raft', async function () {
+    let raftContract: Raft
+    const name = 'Raft'
+    const symbol = 'RAFT'
+    const chainId = 31337
+    const tokenURI = 'blah'
+
+    async function deployContractFixture() {
+        const Raft = await ethers.getContractFactory('Raft')
+        const [owner, addr1] = await ethers.getSigners()
+        const raftContract = await Raft.deploy(owner.address, name, symbol)
+        await raftContract.deployed()
+        return { Raft, raftContract, owner, addr1 }
+    }
+
+    it('should deploy the contract with the right params', async function () {
+        const { raftContract } = await loadFixture(deployContractFixture)
+        const deployedContractName = await raftContract.name()
+        const deployedSymbolName = await raftContract.symbol()
+        expect(deployedContractName).to.equal(name)
+        expect(deployedSymbolName).to.equal(symbol)
+
+        const provider = waffle.provider
+        const network = await provider.getNetwork()
+        const deployedToChainId = network.chainId
+        expect(deployedToChainId).to.equal(chainId)
+    })
+
+    it('should allow owner to mint when minting is paused', async function () {
+        const { raftContract, addr1 } = await loadFixture(deployContractFixture)
```

Comment:
How about we add explicit assertion that the assumed state of the contract is as-needed in order for the test to pass?

for ex,
```typescript
expect(isPaused).to.equal(false)
```